config/routes.rb

```diff
@@ -5,11 +5,14 @@
   get "/submissions/rejected", to: "submissions#rejected"
   get "/submissions/to_rate", to: "submissions#to_rate"
   get "/submissions/results", to: "submissions#results"
+  get "/submissions/confirm", to: "submissions#confirm"
```

Comment:
Initially, I wanted to agree, but then I thought that it's needed for the link in the email, so it's like email confirmation or something. I think `get` is ok.